db/migrate/20170909210000_create_legislator.rb

```diff
@@ -0,0 +1,18 @@
+class CreateLegislator < ActiveRecord::Migration[5.0]
+  def change
+    create_table :legislators, id: :uuid do |t|
+      t.column :external_id, :integer, null: false
```

Comment:
since we can't set this from the openstates service, this column can't be `null: false`